modules/frontend/tracebyidsharding.go

```diff
@@ -191,6 +221,15 @@ func (s shardQuery) RoundTrip(r *http.Request) (*http.Response, error) {
 	}, nil
 }
 
+// recordTraceByIDMetrics records throughput and SLO metrics
+func (s shardQuery) recordMetrics(tenantID string, status int, reqTime time.Duration) {
+	// we are within SLO if query returned within DurationSLO seconds
+	// TODO: we don't have throughput metrics for TraceByID yet
+	if reqTime < s.sloCfg.DurationSLO {
+		sloTraceByIDPerTenant.WithLabelValues(tenantID, strconv.Itoa(status)).Inc()
```

Comment:
oh, i see. you have two different metrics here. i was confused by the similar structures. my personal .02 is that this SLO metric should mirror this metric:

https://github.com/grafana/tempo/blob/main/modules/frontend/frontend.go#L61

it should be named similarly so they are discoverable together and have basically the same labels so it's to combine them in promql math queries.